src/Microsoft.AspNetCore.Dispatcher.Abstractions/Microsoft.AspNetCore.Dispatcher.Abstractions.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Description>Abstractions for dispatching.</Description>
```

Comment:
Can we also add some tags while we're at it?

Some examples:
* https://github.com/aspnet/Routing/blob/dev/src/Microsoft.AspNetCore.Routing/Microsoft.AspNetCore.Routing.csproj#L4-L11
* https://github.com/aspnet/Routing/blob/dev/src/Microsoft.AspNetCore.Routing.Abstractions/Microsoft.AspNetCore.Routing.Abstractions.csproj#L4-L11

Reply:
(These are for @rynowak to provide.)